modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsColumnImageFieldMerge.php

```diff
@@ -10,6 +10,25 @@
 /**
  * Process Plugin to field merge for Column Image paragraphs.
  *
+ * Available configuration keys
+ * - caption: The field ID for the source image caption.
+ * - credit: The field ID for the source image credit.
+ * - format: The destination filter format for the migrated text.
+ *
+ * Examples:
+ *
+ * Consider a paragraph item migration, where you want to merge the source
+ * caption and credit fields into the destination paragraph.
+ * @code
+ * process:
+ *   field_az_text_area:
+ *     plugin: paragraphs_column_image_field_merge
+ *     source: '@temp_photo'
+ *     caption: field_uaqs_caption_text
+ *     credit: field_uaqs_image_credit
+ *     format: 'az_standard'
+ * @endcode
+ *
```

Comment:
I'm not sure that this migration is actually correct, since we added caption and credit to the media migration here:
https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_migration/migrations/migrate_plus.migration.az_media.yml#L49-L66

Additionally, the az_large view mode for images contains credit and caption.
https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_media/config/install/core.entity_view_display.media.az_image.az_large.yml#L78-L91